pyaedt/misc/aedtlib_personalib_install.py

```diff
@@ -12,27 +12,15 @@
     "..",
 )
 sys.path.append(os.path.join(pyaedt_path, ".."))
-student_version = False
-if len(sys.argv) < 2:
-    version = "2022.2"
-elif sys.argv[1].endswith("sv"):
-    v = sys.argv[1][:-2]
-    version = "20" + v[-3:-1] + "." + v[-1:]
-    student_version = True
-else:
-    v = sys.argv[1]
-    version = "20" + v[-3:-1] + "." + v[-1:]
-sys_lib = True
-if len(sys.argv) == 3:
-    sys_lib = True if sys.argv[2] == "1" else False
+
 
 pid = 0
 
 
-def main():
+def add_pyaedt_to_aedt(aedt_version, is_student_version=True, use_sys_lib=False):
```

Comment:
Maybe the default value for the `is_student_version` parameter should be `False` as most users are not using student version.
```suggestion
def add_pyaedt_to_aedt(aedt_version, is_student_version=False, use_sys_lib=False):
```